### PR TITLE
Add K8 support to IBM MQ Mixin

### PR DESCRIPTION
**Pull request by BominRahmani**
This PR adds k8's support to the IBM MQ mixin.

### CODE REVIEW

File: ibm-mq-mixin/config.libsonnet [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {
   _config+:: {
+    enableMultiCluster: false,
+    ibmmqSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
     dashboardTags: ['ibm-mq-mixin'],
     dashboardPeriod: 'now-1h',
     dashboardTimezone: 'default',
     dashboardRefresh: '1m',
+    logExpression: if self.enableMultiCluster then '{job=~"$job", cluster=~"$cluster"'
+    else '{job=~"$job"',
```
Comment (gaantunes) [inline ibm-mq-mixin/config.libsonnet:10]: I would rather have the `{` inside the dashboard code for consistency.

Comment (Dasomeone) [reply ibm-mq-mixin/config.libsonnet:10]: Agreed, would make it a little simpler to maintain

Comment (BominRahmani) [reply ibm-mq-mixin/config.libsonnet]: Agreed! Done! @Dasomeone @gaantunes

**Review by Dasomeone — APPROVED** [review]

**Review by gaantunes — APPROVED** [review]
LGTM.

**Pull request merged**